lib/main/atom/commands/commands.ts

```diff
@@ -99,6 +100,23 @@ export function registerCommands() {
 
         parent.build({ filePath: filePath }).then((resp) => {
             buildView.setBuildOutput(resp.buildOutput);
+            let editorsObserver = atom.workspace.observeTextEditors((editor) => {
```

Comment:
You can use `atom.workspace.getTextEditors() ` instead of subscribing and unsubscribing :wink: